### ISSUE TITLE
Rails/ActiveRecordOverride crashing on modules

### ISSUE THREAD

**Issue by ghiculescu** [rubocop-hq/rubocop#6943]
`Rails/ActiveRecordOverride` crashes on the following code:

```ruby
module Foo
  def bar
    true
  end

  def update
    true
  end
end
```

```
Scanning test.rb
An error occurred while Rails/ActiveRecordOverride cop was inspecting test.rb:6:2.
undefined method `const_name' for nil:NilClass
/Users/alex/.rvm/gems/ruby-2.5.3/gems/rubocop-0.67.2/lib/rubocop/cop/rails/active_record_override.rb:42:in `on_def'
```

Note that it does not crash if the module only contains 1 method.

This is the same root cause as https://github.com/rubocop-hq/rubocop/issues/6888 but I logged it separately as it impacts modules, which IMO this cop shouldn't check.

--------

## Expected behavior

Don't crash, but record a lint error.

## Actual behavior

Crash.

## Steps to reproduce the problem

Code above.

## RuboCop version

0.67.2 (using Parser 2.6.2.1, running on ruby 2.5.3 x86_64-darwin15)

**Comment by andreasklinger** [comment]
Same error
```
    rubocop (0.67.2)
    parser (2.6.2.1)
    ruby 2.4.5p335 (2018-10-18 revision 65137) [x86_64-darwin18]
```

**Comment by andreasklinger** [comment]
happens on all rubocop version - seems to be the cop itself

i didn't look into this properly now but this workaround might help someone landing in this issue
add this to rubocop.yml
```
Rails/ActiveRecordOverride:
  Enabled: false
```

**Comment by tiagotex** [comment]
I also found the same problem, modules don't have a parent class so 

```ruby
```

**Comment by tiagotex** [comment]
Well, it seems like this is a duplicate of #6888 and it is already solved on master (#6889) 😄

**Comment by ghiculescu** [comment]
Thanks for that, any ideas when that’ll be on rubygems?
